src/constants/mod.rs

```diff
@@ -18,6 +18,10 @@ pub const POOLID_SIZE_BITS: usize = 24;
 pub const POLY_1305_TAG_SIZE: usize = 16;
 pub const U256_SIZE:usize = 32;
 
+// TODO: check params
+pub const DAY_SIZE_BITS: usize = 24;
+pub const TURNOVER_SIZE_BITS: usize = 72;
```

Comment:
How does particular bit sizes choice impact the resource requirements during proving/verifying phases?

Reply:
Increasing DAY_SIZE_BITS or TURNOVER_SIZE_BITS by 8 bits adds 32 additional gates. This is not very significant, since the total number of gates is about 203k. This should also not affect verification phase.